### DISCUSSION TITLE
Configuration

### DISCUSSION THREAD

**Discussion by ghost** [hassio-addons/addon-home-panel#184] [Q&A]
Hi, I may be missing something to configure the addon. My problem is addon works but When I try to change the primary color or anything else in the configuration page nothing happens. I have even started homeassistant in Virtualbox new fresh copy but the same thing. Also, tried different computers with fresh install homeassistant -nothing. Then I set up ssl and domain and activated https:// again same problem still. Chrome is showing in console this error:

![homepanel](https://user-images.githubusercontent.com/10806497/213071222-71c2b726-12d5-4cbb-834d-b79a14aa61fd.png)

**Comment by scoleman43** [comment]
I am having the same issue. On a brand new install of Home Assistant & Home Panel.

Add-on version: 2.5.0
 You are running the latest version of this add-on.
 System: Home Assistant OS 9.4  (amd64 / generic-x86-64)
 Home Assistant Core: 2023.1.5
 Home Assistant Supervisor: 2022.12.1

**Comment by Phyxx001** [comment]
Same with me. And googleing it reveals that a good number of people is seeing this - without any clue why. Any update on this?
